sql/table.go

```diff
@@ -88,12 +89,18 @@ func makeTableDesc(p *parser.CreateTable) (structured.TableDescriptor, error) {
 			}
 			if d.PrimaryKey {
 				index.Name = "primary"
+				desc.Indexes = append([]structured.IndexDescriptor{index}, desc.Indexes...)
+			} else {
+				desc.Indexes = append(desc.Indexes, index)
```

Comment:
It might be easier to sort the indexes by name, but ensure that `"primary"` sorts first.